src/OpenLoco/GameCommands/Cheat.cpp

```diff
@@ -0,0 +1,223 @@
+        }
+
+        static uint32_t companyRatings(bool absolute, int32_t value)
+        {
+            auto companyId = CompanyManager::getControllingId();
+
+            for (auto& town : TownManager::towns())
+            {
+                // Does this town have a rating for our company?
+                if (!(town.companies_with_rating &= (1 << companyId)))
+                    continue;
+
+                int16_t newRanking{};
+                if (absolute)
+                {
+                    newRanking = value * max_company_rating;
+                }
+                else
+                {
+                    newRanking = town.company_ratings[companyId] + max_company_rating;
+                    newRanking *= 1.0f + (1.0f / value);
+                    newRanking -= max_company_rating;
+                }
+
+                // Set the new rating.
+                town.company_ratings[companyId] = std::clamp<int16_t>(newRanking, min_company_rating, max_company_rating);
+            }
+
+            return 0;
+        }
+
+        static uint32_t switchCompany(CompanyId_t targetCompanyId)
+        {
+            auto ourId = CompanyManager::getControllingId();
+            auto otherId = CompanyManager::getSecondaryPlayerId();
+
+            // Already controlling the target company?
+            if (targetCompanyId == ourId)
+                return 0;
+
+            // Is the other player controlling the target company? Swap companies.
+            if (targetCompanyId == otherId)
+            {
+                CompanyManager::setSecondaryPlayerId(ourId);
+                CompanyManager::setControllingId(otherId);
+                return 0;
+            }
+
+            // Change control over to the other company.
+            CompanyManager::setControllingId(targetCompanyId);
+            return 0;
+        }
+
+        static uint32_t toggleBankruptcy(CompanyId_t targetCompanyId)
+        {
+            auto company = CompanyManager::get(targetCompanyId);
+            company->challenge_flags ^= CompanyFlags::bankrupt;
+            return 0;
+        }
+
+        static uint32_t toggleJail(CompanyId_t targetCompanyId)
+        {
+            auto company = CompanyManager::get(targetCompanyId);
+            company->jail_status = 30;
+            return 0;
+        }
+
+        static uint32_t vehicleReliability(int32_t newReliablity)
+        {
+            auto ourCompanyId = CompanyManager::updatingCompanyId();
+
+            for (auto vehicle : EntityManager::VehicleList())
+            {
+                if (vehicle->owner != ourCompanyId)
+                    continue;
+
+                Vehicles::Vehicle train(vehicle);
+                // train.head->reliability = newReliablity;
+                // train.veh1->reliability = newReliablity;
+                train.veh2->reliability = newReliablity;
+                // train.tail->reliability = newReliablity;
+
+                for (auto& car : train.cars)
```

Comment:
Pretty sure only the front bogie of each car has a reliability.
So you don't need the nested loop nor the veh2 setting of reliability.